rotkehlchen/chain/ethereum/modules/compound/decoder.py

```diff
@@ -122,6 +124,49 @@ def _decode_redeem(
         maybe_reshuffle_events(out_event=out_event, in_event=in_event, events_list=decoded_events)
         return None, []
 
+    def _decode_borrow_and_repay(
+            self,
+            tx_log: EvmTxReceiptLog,
+            decoded_events: list[HistoryBaseEntry],
+            compound_token: EvmToken,
+    ) -> tuple[Optional[HistoryBaseEntry], list[ActionItem]]:
+        """
+        Decode borrow and repayments for compound tokens
+        """
+        underlying_token = get_crypto_asset_by_symbol(
```

Comment:
Won't it be better to check `underlying_tokens` of the compound token?

Reply:
The reason is that for example the CUSDC token (0x39AA39c021dfbaE8faC545936693aC917d5E7563) doesn't have USDC as underlying token in our database and we use the symbol in the token to know what token is the underlying one. We can change it but I would do in another PR. Also not 100% sure if they have a registry or something to keep it up to date

Reply:
Yeah, okay, checking underlying tokens is just cleaner, but the current approach should be okay too I guess.

The only question is whether cETH can appear here? If yes, then the cast to evm token that is happening below will be wrong.